rpc/tendermint/test/common.go

```diff
@@ -32,7 +32,7 @@ func TestWrapper(runner func() int) int {
 func DebugMain() {
 	t := &testing.T{}
 	TestWrapper(func() int {
-		testBroadcastTx(t, "HTTP")
```

Comment:
DebugMain is my convenience method that allows me to run delve, it's not part of normal testing.